src/screens/Nearby/section-items/line.tsx

```diff
@@ -0,0 +1,225 @@
+import Warning from '../../../assets/svg/situations/Warning';
+import {screenReaderPause} from '../../../components/accessible-text';
+import Button from '../../../components/button';
+import {
+  SectionItem,
+  useSectionItem,
+  useSectionStyle,
+} from '../../../components/sections/section-utils';
+import ThemeText from '../../../components/text';
+import {ThemedTransportationIcon} from '../../../components/transportation-icon';
+import {StyleSheet} from '../../../theme';
+import colors from '../../../theme/colors';
+import {
+  dictionary,
+  Language,
+  NearbyTexts,
+  useTranslation,
+} from '../../../translations';
+import {
+  formatToClock,
+  formatToClockOrRelativeMinutes,
+  isInThePast,
+  isRelativeButNotNow,
+} from '../../../utils/date';
+import insets from '../../../utils/insets';
+import {hasNoDeparturesOnGroup, isValidDeparture} from '../utils';
+
+export type LineItemProps = SectionItem<{
+  group: DepartureGroup;
+  accessibility?: AccessibilityProps;
+}>;
+export default function LineItem({
+  group,
+  accessibility,
+  ...props
+}: LineItemProps) {
+  const {contentContainer, topContainer} = useSectionItem(props);
+  const sectionStyle = useSectionStyle();
+  const styles = useItemStyles();
+  const navigation = useNavigation<NearbyScreenNavigationProp>();
+  const {t} = useTranslation();
+
+  if (hasNoDeparturesOnGroup(group)) {
+    return null;
+  }
+
+  const title = `${group.lineInfo?.lineNumber} ${group.lineInfo?.lineName}`;
+
+  const onPress = (departure: DepartureTime) => {
+    navigation.navigate('DepartureDetailsModal', {
+      title,
+      serviceJourneyId: departure.serviceJourneyId!,
+      date: departure.aimedTime,
+      fromQuayId: group.lineInfo?.quayId,
+    });
+  };
+
+  // we know we have a departure as we've checked hasNoDeparturesOnGroup
+  const nextValids = group.departures.filter(isValidDeparture);
+  const firstResult = nextValids[0]!;
+
+  return (
+    <View style={[topContainer, {padding: 0}]}>
+      <View style={[topContainer, sectionStyle.spaceBetween]}>
+        <TouchableOpacity
+          style={styles.lineHeader}
+          containerStyle={contentContainer}
+          onPress={() => onPress(firstResult)}
+          hitSlop={insets.symmetric(12, 0)}
+          accessibilityRole="button"
+          accessibilityHint={
+            t(NearbyTexts.results.lines.lineNameAccessibilityHint) +
+            screenReaderPause
+          }
+          accessibilityLabel={getAccessibilityTextFirstDeparture(
+            title,
+            nextValids,
+            t,
+          )}
+        >
+          <View style={styles.transportationMode}>
+            <ThemedTransportationIcon mode={group.lineInfo?.transportMode} />
+          </View>
+          <ThemeText>{title}</ThemeText>
+        </TouchableOpacity>
+      </View>
+      <ScrollView
+        horizontal
+        contentContainerStyle={styles.scrollContainer}
+        accessibilityElementsHidden
```

Comment:
Also need to provide the Android variant, `importantForAccessibility="no"

Reply:
Ah. Thanks 😍  (one would think this could be patched internally...)

Reply:
Should it be `importantForAccessibility="no-hide-descendants"` ?

Reply:
Yes, doesn't make sense that they are two different properties, but perhaps to support the no/no-hide-descendants difference on Android . "no" hides only the current element, "no-hide-descendats" hides all children. The latter is an easy "go to", I think that's almost always what we'd really want to do.